python/taichi/lang/__init__.py

```diff
@@ -256,11 +252,14 @@ def supported_archs():
     wanted_archs = os.environ.get('TI_WANTED_ARCHS', '').split(',')
     # Note, ''.split(',') gives you [''], which is not an empty array.
     wanted_archs = list(filter(lambda x: x != '', wanted_archs))
-    if len(wanted_archs):
+    excluded_archs = list(filter(lambda x: x.startswith('^'), wanted_archs))
```

Comment:
Please, revert this part, do this in a different PR and use `ti test -a x64` for now. The correctness of this part is not trivial and thereby needs discussions. In this PR I'd rather focus on GLAD.